python/cuml/dask/ensemble/randomforestregressor.py

```diff
@@ -74,7 +74,7 @@ class RandomForestRegressor(BaseRandomForestModel, DelayedPredictionMixin,
     split_criterion : int (default = 2)
         The criterion used to split nodes.
         0 for GINI, 1 for ENTROPY,
-        2 for MSE, 3 for MAE and 4 for CRITERION_END.
+        2 for MSE, 3 for MAE and 4 for POISSON
```

Comment:
Eventually I want to use strings here like sklearn. Changes to the python code base are for another day though.